pkg/sink/sink.go

```diff
@@ -354,3 +360,20 @@ func (r Sink) CreateResources(triggerNS, sa string, res []json.RawMessage, trigg
 	}
 	return nil
 }
+
+// extendBodyWithExtensions merges the extensions into the given body.
+func extendBodyWithExtensions(body []byte, extensions map[string]interface{}) ([]byte, error) {
+	newBody := body
```

Comment:
I don't think you need to reassign body here?

Reply:
done